data/json/itemgroups/SUS/fridges.json

```diff
@@ -844,89 +844,63 @@
     "subtype": "collection",
     "entries": [
       {
-        "distribution": [
-          { "item": "apple", "prob": 25 },
-          { "item": "orange", "prob": 25 },
-          { "item": "banana", "prob": 25 },
-          { "item": "pear", "prob": 10 },
+        "collection": [
```

Comment:
Because the parent group is a collection, you don't need these to be separate groups, they can be just on the base level. The only time you'd want to make these nested as separate collections is if you wanted to do something like "there's a fifty percent chance of having any fruit in the fridge, and if there is fruit, it will be selected from this list". Which I don't think makes sense in this context.